apps/package.json

```diff
@@ -32,6 +32,7 @@
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "joi": "^17.6.0",
+    "lodash.get": "^4.4.2",
```

Comment:
Since `lodash` is likely already an internal dependency, maybe it's better to just import the entire `lodash` package?

When compiled it will be tree shaken out, giving the same benefit of using `lodash.get.

Reply:
Hmm I don't see lodash being used anywhere in jellyfish right now (ctrl shift f 'lodash'), don't see it in root package.json either

Reply:
`npm list lodash`, it's a internal dependencies, when you add it to the root `apps/package.json`, it will be deduped